pkgs/development/python-modules/vllm/default.nix

```diff
@@ -52,22 +72,23 @@ let
     owner = "NVIDIA";
     repo = "cutlass";
     rev = "refs/tags/v3.5.0";
-    sha256 = "sha256-D/s7eYsa5l/mfx73tE4mnFcTQdYqGmXa9d9TCryw4e4=";
+    hash = "sha256-D/s7eYsa5l/mfx73tE4mnFcTQdYqGmXa9d9TCryw4e4=";
   };
 in
 
 buildPythonPackage rec {
   pname = "vllm";
-  version = "0.5.3.post1";
+  version = "0.6.0";
   pyproject = true;
 
   stdenv = if cudaSupport then cudaPackages.backendStdenv else args.stdenv;
 
   src = fetchFromGitHub {
     owner = "vllm-project";
-    repo = pname;
+    repo = "vllm";
```

Comment:
Why?

Reply:
The use of `pname` in `fetchFromGitHub` is discouraged because it could lead to silently introducing error.
I have taken the habit to correct those things when I come accross them/